jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1070,6 +1070,14 @@ properties:
                 title: Dark reference file name
                 type: string
                 fits_keyword: R_DARK
+          darkMIRI:
```

Comment:
Don't forget to remove line 1073 as well, which is the starting line of the new meta data entry that had been created.